envs/conda-env.yml

```diff
@@ -0,0 +1,132 @@
+name: snakemake
+channels:
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
That's an impressive list of dependencies, both Conda and PIP. I assume that it was generated by `conda list --export` + `pip freeze` (or something along those lines). Pinning all of them to specific versions has advantages; for one, it ensures maximum reproducibility. However, do we have a plan on how to re-generate/update this list in the future?

Reply:
At some point we certainly need to do something about this list for sure. However this is not an urgent fix, I would rather open a ticket with a list of issues we need to resolve later. I see this is not a blocker for the merge.

Reply:
Added to the follow-up ticket.